tasks.py

```diff
@@ -1,36 +1,64 @@
+import shlex
 import subprocess
+import sys
 from pathlib import Path
+from typing import Optional
 
 import invoke
-from invoke import task
+from invoke import call, task
 
-CURDIR: Path = Path(__file__).absolute().parent
+ROOT: Path = Path(__file__).absolute().parent
+DOCS_IGNORE = ["devutils"]
 
 
 @task
-def poetry_lock(ctx: invoke.Context) -> None:
-    """Runs 'poetry lock' on all projects with a pyproject.toml."""
+def install(
```

Comment:
Decided to combine 3 overlapping commands into one:
- `poetry-lock`, `poetry-update` -> `inv install -u`
- `poetry-install` -> `inv install`